app/controllers/ticket_comments_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class TicketCommentsController < ApplicationController
+  include Rails::Pagination
```

Comment:
Now this inclusion is useless :upside_down_face:

Reply:
Whoopsieee, removed.